docs/data-sources/domain.md

```diff
@@ -177,15 +176,28 @@ Read-Only:
 - `user_id` (String) UUID of the VASA storage user
 - `vasa_provider_id` (String) UUID of the VASA storage provider
 
-
-
-<a id="nestedatt--nsx_cluster_ref"></a>
-### Nested Schema for `nsx_cluster_ref`
-
 Read-Only:
 
 - `id` (String) NSX Manager cluster ID
+- `nsx_manager_node` (List of Object) (see [below for nested schema](#nestedobjatt--nsx_configuration--nsx_manager_node))
 - `vip` (String) Virtual IP (VIP) for the NSX Manager cluster
 - `vip_fqdn` (String) Fully qualified domain name of the NSX Manager cluster VIP
 
+<a id="nestedobjatt--nsx_configuration--nsx_manager_node"></a>
+### Nested Schema for `nsx_configuration.nsx_manager_node`
+
+Read-Only:
+
+- `dns_name` (String) Fully qualified domain name of the NSX Manager appliance, e.g., sfo-w01-nsx01a.sfo.rainpole.io
```

Comment:
agree!

Reply:
Sure! I've used "dns_name" as that is what the API uses, see:
https://developer.vmware.com/apis/vcf/4.5.0/data-structures/NetworkDetailsSpec/

If I rename that attribute from "dns_name" to "fqdn" that will be a breaking change in the domain resource, since I'm reusing the code that defines the attribute schema.

Reply:
Given this is already a breaking change, I think it's ok to fix things for consistency and correctness.

Reply:
I agree!